package.json

```diff
@@ -54,7 +54,7 @@
     "test": "jest",
     "clean:allureFolders": "rm -R -f allure-reports/* && rm -R -f allure-results/*",
     "beforeIntTest": "npm run clean:allureFolders",
-    "test:integration": "jest --config jest.integration.config.ts",
+    "test:integration": "jest --config jest.integration.config.ts -i",
```

Comment:
Is this flag necessary? Ideally we'd keep using a worker pool to run then tests rather than serialising them

Reply:
wanted to run the test sequentially as we are deleting the S3 directory in view tests so when running the test parallel other tests will be interrupted. We can enable to parallel once we implement the clean up for each test